beacon-chain/blockchain/process_block.go

```diff
@@ -405,6 +403,25 @@ func (s *Service) onBlock(ctx context.Context, signed interfaces.SignedBeaconBlo
 	return nil
 }
 
+func (s *Service) tryPublishLightClientFinalityUpdate(ctx context.Context, signed interfaces.SignedBeaconBlock, finalized *forkchoicetypes.Checkpoint, postState state.BeaconState) {
+	if finalized.Epoch > s.lastPublishedLightClientEpoch {
```

Comment:
This can be flipped to reduce nesting:

if finalized.Epoch <= s.lastPublishedLightClientEpoch {
  return
}
// ...

Reply:
Good point.